functions.php

```diff
@@ -14,13 +14,13 @@
 
 // Bail if requirements are not met.
 if ( version_compare( $GLOBALS['wp_version'], WP_RIG_MINIMUM_WP_VERSION, '<' ) || version_compare( phpversion(), WP_RIG_MINIMUM_PHP_VERSION, '<' ) ) {
-	require get_template_directory() . '/inc/back-compat.php';
+	require get_stylesheet_directory() . '/inc/back-compat.php';
```

Comment:
If we want to make sure the WP Rig files are always called, we might have to replace all the usage of `get_template_directory()` with `dirname( __FILE__ )`.